src/Statistics.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  color: #383854;
+  margin: 4rem 0 2rem 0;
+  font-size: 2rem;
+`;
+
+const ImgContainer = styled.div`
+  color: white;
+  border: #383854 solid 0.5rem;
+  border-radius: 1rem;
```

Comment:
```
border: #383854 solid 0.5rem;
 border-radius: 1rem;
```
instead of putting them in ImgContainer put them in Container
and add a title style that wll have a background of 100% width in the container